portlets/src/main/webapp/vue-app/engagement-center/components/rules/RuleDetailDrawer.vue

```diff
@@ -199,9 +200,7 @@ export default {
       return this.program?.audienceId;
     },
     announceDisabled() {
-      return !this.validInput
-          || !this.comment
-          || !this.comment.length;
+      return !this.validInput || !this.comment || !this.comment.length || this.canAnnounce;
```

Comment:
Loading effect is sufficient to disable the button in fact. The `disable` value should be different from `loading` value of button.

```suggestion
      return !this.validInput || !this.comment || !this.comment.length;
```

please change the button definition to use `v-btn` instead with current definition : 
```
<v-btn
  :loading="sending"
  :disabled="buttonDisabled"<!-- to rename variable IMO to be more consistent -->
```

Reply:
done